service/controller/chart/resource/status/create.go

```diff
@@ -35,6 +35,8 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	releaseName := key.ReleaseName(cr)
 	r.logger.Debugf(ctx, "getting status for release %#q", releaseName)
 
+	r.logger.Debugf(ctx, "the controller context for %#q is: %#q", releaseName, cc)
```

Comment:
Is it useful when debugging the Chart CR?

Reply:
Nope, it will end up in the status reason now, cleaned it up.